<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instWizard</class>
 <widget class="QDialog" name="instWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="headerLabel">
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Installation Wizard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="deviceLabel">
        <property name="text">
         <string>Select Device</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="deviceComboBox"/>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="wikiLookupButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Start search from</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="searchPath">
        <property name="text">
         <string>/home</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="fileSelectButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="possibleHitsLabel">
        <property name="text">
         <string>Possible Hits:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>266</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="startSearchButton">
        <property name="text">
         <string>search driver</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QTableWidget" name="resultTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="driverLabel">
        <property name="text">
         <string>Driver : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="driverEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="cancelInstButton">
     <property name="text">
      <string>Cancel Installation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>229</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="startInstButton">
     <property name="text">
      <string>Start Installation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QPlainTextEdit" name="comment">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>The wizard will help you to locate the appropriate inf-file and the according sys-file for the installation of the windows driver.

First select the proper device from the device list.

By pressing the button right to the list, the offline file from the ndiswrapper wiki (card known to work) is displayed. You maybe find helpful information for your hardware.

Next you have to tell the wizard from where to start the search run. If you got a dual boot system and the appropriate driver is already installed in the windows system you may mount the windows partition and give the mountpoint as starting point for the search run.

The result of the search run is displayed in the list below.
Select a driver from the list and press the install button.

The wizard even tries to find the correct sys-file if it is not located in the same folder as the inf-file.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>wikiLookupButton</sender>
   <signal>clicked()</signal>
   <receiver>instWizard</receiver>
   <slot>slot_wikiLookupButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>725</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileSelectButton</sender>
   <signal>clicked()</signal>
   <receiver>instWizard</receiver>
   <slot>slot_fileSelectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startSearchButton</sender>
   <signal>clicked()</signal>
   <receiver>instWizard</receiver>
   <slot>slot_startSearchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelInstButton</sender>
   <signal>clicked()</signal>
   <receiver>instWizard</receiver>
   <slot>slot_cancelInstButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startInstButton</sender>
   <signal>clicked()</signal>
   <receiver>instWizard</receiver>
   <slot>slot_startInstButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resultTable</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>instWizard</receiver>
   <slot>slot_resultTableClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_fileSelectButtonClicked()</slot>
  <slot>slot_wikiLookupButtonClicked()</slot>
  <slot>slot_startSearchButtonClicked()</slot>
  <slot>slot_resultTableClicked()</slot>
  <slot>slot_cancelInstButtonClicked()</slot>
  <slot>slot_startInstButtonClicked()</slot>
 </slots>
</ui>
